frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -163,9 +153,16 @@ export default ({
   // showAddReviewButton,
   linkSubmission,
   formErrors,
-}) => {
+}) {
   const classes = useStyles();
 
+  let contentLink, contentItemId;
+  // sometimes card is undefined
+  if (card !== undefined) {
+    contentLink = card.contentItemUrl;
```

Comment:
why is this sometimes called contentItemUrl and other times it's called contentLink? Is there a reason to call one thing by two names?